### PR TITLE
Feature: count 

### PR DESCRIPTION
**Pull request by AngelGzS**
Fix for #94

### CODE REVIEW

**Comment by AngelGzS** [comment]
Someone knows how to fix it?

**Comment by theashraf** [comment]
Try running `yarn bootstrap` before running `yarn test` again.

**Comment by AngelGzS** [comment]
> Try running `yarn bootstrap` before running `yarn test` again.

Thank you!

File: packages/count/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## `nameOfTheModule`
```
Comment (vorillaz) [inline packages/count/README.md]: Oops.

File: packages/count/src/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * count('Foo') // returns 3
+ * count('Foo bar') // returns 7
+ * count('Foo bar  ', char => char != ' ') //returns 6
+ * 
+ */
+const count = (text, validator) => {
+  let strLen = text ? text.length : 0;
+
+  if (typeof validator == 'function')
+  {
```
Comment (vorillaz) [inline packages/count/src/index.js]: I think prettier missed that line.

File: packages/count/src/index.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ * @description This is a sample description
+ * @param {String} text
+ * @param (Function) validator, validates if a character should be count or not
+ * @example
+ * count('b') // returns 1
+ * count('Foo') // returns 3
+ * count('Foo bar') // returns 7
+ * count('Foo bar  ', char => char != ' ') //returns 6
+ * 
+ */
```
Comment (vorillaz) [inline packages/count/src/index.js]: I would be suggesting to exit early on the method or even using a default method as the validator.
```
const count = (text, validator = () => true) => {}
```

File: packages/count/test/index.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  it('empty string given and no validator for second parameter', () => {
+    expect(count('')).toBe(0);
+  });
+
+  it('no empty string given for first and no validator for second parameter ', () => {
+    expect(count('Foo bar')).toBe(7);
+  });
+
+  it('no empty string given for first and  validator for no spaces as second parameter', () => {
+    expect(count('Foo bar   ', char => char != ' ')).toBe(6);
```
Comment (vorillaz) [inline packages/count/test/index.js]: Please use `!==` as a comparator.

File: packages/count/src/index.js (hunk 3 above)
Comment (AngelGzS) [reply packages/count/src/index.js]: I like validator = () => true) as default method

**Pull request merged**